hibp/client.go

```diff
@@ -62,3 +62,31 @@ func createClient(ctx context.Context, apiKey string) *hibp.Client {
 	cl := hibp.New(clientOptions...)
 	return &cl
 }
+
+// Create a client without an API key
+// Certain tables like hibp_password do not require an API key
```

Comment:
Are there any other tables that don't require an API key?

For those that don't, if we introduce this change, we should update all of the table docs with a line mentioning if an API key is required or not.

Reply:
Referring to the [HIBP plugin docs here](https://hub.steampipe.io/plugins/turbot/hibp/tables) there are only 4 tables, of those tables only the pwned passwords API doesn't require a key whereas breaches and pastes do require a key.